dataland-frontend/tests/e2e/specs/search-company-framework-data/SearchCompaniesForFrameworkDataDropdownFilter.ts

```diff
@@ -103,36 +104,49 @@ describe("As a user, I expect the search functionality on the /companies page to
     "Checks that the sector filter synchronises between the search bar and the drop down and works",
     { scrollBehavior: false },
     () => {
-      const demoCompanyToTestFor = companiesWithEuTaxonomyDataForNonFinancials[0].companyInformation;
+      const demoCompanyToTestFor = assertDefined(
+        companiesWithEuTaxonomyDataForNonFinancials.find((it) => it.companyInformation.sector !== undefined)
+      ).companyInformation;
       const demoCompanyWithDifferentSector = companiesWithEuTaxonomyDataForNonFinancials.find(
-        (it) => it.companyInformation.sector !== demoCompanyToTestFor.sector
+        (it) =>
+          it.companyInformation.sector !== demoCompanyToTestFor.sector && it.companyInformation.sector !== undefined
       )!.companyInformation;
 
+      expect(demoCompanyToTestFor.sector).to.not.be.undefined;
```

Comment:
Entweder sollte das `expect()` weg, weil ihr in der `.find()` Methode oben schon sichergestellt habt, dass `sector` nicht `undefined` ist, oder wenn ihr trotzdem nochmal auf Nummer sicher gehen wollt, dann dieses `expect` auch für beide demoCompanies machen, oder?

Reply:
Naja, wenn `find` nichts findet. Ist das Result `undefined`. Wir wollen ja aber, dass etwas gefunden wird. Daher das `expect`. Jo, bei der anderen spekulieren wir irgendwie mehr. Hängt immer von den FakeFixtures ab. Ich habs hinzugefügt. To be pushed.

Reply:
Stimmt hast recht.
Dann lass uns bei beiden nicht spekulieren würde ich sagen und zweimal das "expect" einbauen.

Reply:
👍